1-JavaScript/Week4/homework/ex3-lemonAllergy.test.js

```diff
@@ -26,26 +26,28 @@ const fruitBasket = [
 ];
 
 // ! Function under test
-function sanitizeFruitBasket(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function sanitizeFruitBasket(basket, unwantedItem) {
+  const sanitizedBasket = basket.filter((item) => !(item === unwantedItem));
```

Comment:
Think this would be clearer
```suggestion
  const sanitizedBasket = basket.filter((item) => item !== unwantedItem);
```

Reply:
of course!